lesson-4/index.html

```diff
@@ -4,25 +4,82 @@
     <meta charset="UTF-8">
     <title>Title</title>
     <link rel="stylesheet" href="css/style.css">
+
 </head>
 <body>
-    <p class="title">What do we offer</p>
-    <hr>
-    <div class="container-support">
-        <img src="images/Support.svg" alt="Support 24/7">
-        <p>Support 24/7</p>
+    <div class="container-head">
+        <header>
+            <div class="logo">
+                <img src="images/logo.svg" alt="logo">
+            </div>
+            <div class="nav">
+                <span>Stays</span>
+                <span>Attraction</span>
+                <img src="images/Night.svg" alt="night">
+                <img src="images/AccountCircle.svg" alt="account">
```

Comment:
please center vertically

Reply:
make the elements as links and add an orange stick to them at the bottom using `:hover`